app/views/site/genome/_genome.html.erb

```diff
@@ -13,10 +13,12 @@ See partials referenced below and scp-igv.js for larger bodies of code.
     // Intercept requests to add bearer token, enabling direct load of files from GCS
     var originalFetch = window.fetch;
     window.fetch = function () {
-      var myHeaders = new Headers({
-        'Authorization': 'Bearer ' + accessToken
-      });
-      arguments[1] = {headers: myHeaders};
+      if (arguments[0].includes('https://www.googleapis.com')) {
+        var myHeaders = new Headers({
+          'Authorization': 'Bearer ' + accessToken
+        });
+        arguments[1] = {headers: myHeaders};
+      }
```

Comment:
wow, I did not realize we were overriding window.fetch in SCP, although I think understand why.     Can you add a comment around this?

Reply:
Done in e04553e.  I've also added a ticket for myself next sprint to remove this monkey patching of `fetch` -- SCP-2451.